src/test/java/cucumber/hooks/Hooks.java

```diff
@@ -49,10 +49,10 @@ public void generateToken() {
     }
 
     @Before(order = 1)
-    public void setUp() {
+    public void setUp() throws Exception {
         requestBuilder
                 .addHeader("Authorization", token)
-                .addBaseUri(getTheBaseUrl());
+                .addBaseUri("https://freeorg01com-dev-ed.my.salesforce.com");
```

Comment:
this value should not be hardcoded. Maybe obtain it from the env file.

Reply:
Ok, already done.